website/app/styles/doc-components/image-caption.scss

```
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// IMAGE-CAPTION
+
+@use "../typography/mixins" as *;
+
+.doc-image-caption {
+    @include doc-font-style-body-small();
```

You can simplify it even further by using the class `doc-text-body-small` in the template
```suggestion
.doc-image-caption {
```

---

Nice, don't even need the mixin! Keeping it simple.